package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "scripts": {
     "linter:fix": "prettier --write '{,!(node_modules|.git|storybook-static)/**/}*.{js,json,jsx,md}'",
-    "linter:check": "eslint src/**/* && eslint stories/*.jsx",
+    "linter:check": "eslint 'src/**/*.{tsx, jsx}' && eslint stories/*.jsx",
```

Comment:
Can you explain this change?

It appears to remove linting for .js and .ts files

Reply:
remove linting ? 🤔 , so the test was failing cause it was not finding the path for components in the subfolders for `Col` and `Row` and thats why I did it like that, any suggestions ?

Reply:
Can you try again? previously this picks up all files and lints what it can including js and ts files. Here you are forcing it to only consider tsx and jsx files